game/PsyDoom/Config/ConfigSerialization_Multiplayer.cpp

```diff
@@ -0,0 +1,83 @@
+#include "ConfigSerialization_Multiplayer.h"
+
+#include "Config.h"
+#include "IniUtils.h"
+
+#include <string>
+
+BEGIN_NAMESPACE(ConfigSerialization)
+
+using namespace Config;
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Config field storage
+//------------------------------------------------------------------------------------------------------------------------------------------
+Config_Multiplayer gConfig_Multiplayer = {};
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Initialize the config serializers for multiplayer related config
+//------------------------------------------------------------------------------------------------------------------------------------------
+void initCfgSerialization_Multiplayer() noexcept {
+    auto& cfg = gConfig_Multiplayer;
+
+    cfg.noFriendlyFire = makeConfigField(
+        "NoFriendlyFire",
+        "Cooperative: Players will not take damage from other players.\n"
+        "\n"
+        "Note: players can still take damage from nearby exploding barrels\n"
+        "and being telefragged by the other player.",
+        gbNoFriendlyFire,
+        false
+    );
+
+    cfg.mpThings = makeConfigField(
+        "MPThings",
+        "Cooperative: This will add things to the map that are tagged for multiplayer only\,\n"
+        "such as: additional monsters\, weapons\, and powerups.\n"
+        "\n"
+        "Note: the additional weapons and powerups are normally only found in deathmatch.\n"
+        "The additional monsters were possibly intended for another game mode that was\n"
+        "never implemented or carried over from PC Doom and never removed.",
+        gbMPThings,
+        0
```

Comment:
Minor: use `false` instead of `0` since it's a boolean value.

Reply:
d25abc25e69f6cbe3842bb138dfab2a89fc208e6